axelor-account/src/main/java/com/axelor/apps/account/web/InvoicingPaymentSituationController.java

```diff
@@ -117,4 +122,18 @@ protected Partner getPartner(
 
     return invoicingPaymentSituation.getPartner();
   }
+
+  @ErrorException
+  public void selectBankDetails(ActionRequest request, ActionResponse response) {
+    Context parentContext = request.getContext().getParent();
+    if (parentContext != null && Objects.equals(parentContext.getContextClass(), Partner.class)) {
+      Partner partner = parentContext.asType(Partner.class);
+      List<Long> bankDetailsIdList =
```

Comment:
You can use the StringHelper.getIdListString here but maybe already done in the parent ticket